cvat/requirements/base.txt

```diff
@@ -1,11 +1,14 @@
-# SHA1:dd0cf6b5e35e6d492749b679dccd43a87410571e
+# SHA1:b61b838ad51f0cd9ded9ce9a1a3cdafff2adf74a
 #
 # This file is autogenerated by pip-compile-multi
 # To update, run:
 #
 #    pip-compile-multi
 #
 -r ../../utils/dataset_manifest/requirements.txt
+--no-binary lxml
+--no-binary xmlsec
```

Comment:
@Marishka17 Question for you: how did you originally regenerate the requirements files in #7845? These `--no-binary` options are not supposed to appear here (since they aren't in base.in), but they do appear if I regenerate the files myself (and here we see that they appeared when @zhiltsov-max did it, too). How did you get them to _not_ appear?

Reply:
@SpecLad, @zhiltsov-max,  Oh, sorry, I've just seen this comment now. I fixed them manually since I didn't have time before my vacation to investigate why `pip-compile-multi` initially added `--no-binary` to `utils/dataset_manifest/requirements.txt` (+ it wasn't the only problem that needed fixing after regenerating the requirements, another one was changing the dash to an underscore in a package dependency name).

Reply:
@Marishka17, so should this change be included in the requirements or not?

Reply:
No, we need `lxml` and `xmlsec` installed from source only in the production env.

Reply:
@SpecLad, it seems like we should remove `all` or remove `production` from `all`. `all` is only useful for development, an for development we use `development` and `testing`.

Reply:
`all` is used to support [`--autoresolve`](https://pip-compile-multi.readthedocs.io/en/stable/features.html#autoresolve-cross-file-conflicts). For this reason, it has to include all other requirements files. I suspect that `--autoresolve` is also the reason why the `--no-binary` flags are propagated to every `.txt` file.

We could remove `--autoresolve`, but IMO it's useful, so I don't want to sacrifice it. What I think we should do is handle this the same way we do `av` - move the `--no-binary` flags from requirements files to the Dockerfile.